src/braket/pennylane_plugin/translation.py

```diff
@@ -226,18 +226,20 @@ def translate_result_type(observable: Observable) -> ResultType:
     Returns:
         ResultType: The Braket result type corresponding to the given observable
     """
-    braket_observable = _translate_observable(observable)
     return_type = observable.return_type
     targets = observable.wires.tolist()
 
+    if return_type is ObservableReturnTypes.Probability:
+        return Probability(targets)
```

Comment:
:+1:

Just a thought, but I assume the plugin doesn't support state return? Otherwise that would need to be fixed as well

Reply:
Doesn't support state return yet; figuring out how to wire that in is something I've queued up.